lib/travis/api/v3/renderer/branch.rb

```diff
@@ -5,7 +5,13 @@ class Renderer::Branch < ModelRenderer
     representation(:additional, :recent_builds)
 
     def recent_builds
-      return model.builds.first(10) if include_recent_builds?
+      if include_recent_builds?
+        V3::Models::Build.where(
+          event_type: 'push',
```

Comment:
how about `api` and `cron` builds? were those included before, too?

Reply:
No, they're omitted from the [scope in the Branch model](https://github.com/travis-ci/travis-api/blob/master/lib/travis/api/v3/models/branch.rb#L5).